WordPress/Docs/NamingConventions/PrefixAllGlobalsStandard.xml

```diff
@@ -0,0 +1,53 @@
+<documentation title="Prefix all Globals">
+    <standard>
+    <![CDATA[
+      All globals must be prefixed with a theme/plugin specific term. These prefixes must be distinct and unique to the plugin/theme, in order to prevent potential conflicts with other plugins and themes.
```

Comment:
Suggestion: it might be an idea to mention that the prefix can be freely chosen and set in a custom ruleset with a link to the docs on that: https://github.com/WordPress/WordPress-Coding-Standards/wiki/Customizable-sniff-properties#naming-conventions-prefix-everything-in-the-global-namespace

Reply:
I'm wondering if it would be a good idea to be more specific about what "globals" means in this context.

The sniff at this moment checks:
* Namespace names
* Class/Interface/Trait names (when not namespaced)
* Functions (not methods)
* Constants declared in the global namespace (i.e. constants declared with `const` in the global namespace or with `define()` anywhere)
* Variable names when declared in the global namespace, including when this is done via a `global` statement.
* Hook names

In the future, the sniff is expected to be expanded to also cover post_type names, taxonomy names and more WP-specific constructs, as well as possibly CSS class names and such.

Reply:
Trying to keep this not TOO huge of an explanation...